### DISCUSSION TITLE
Filter collection on a array property

### DISCUSSION THREAD

**Discussion by kingRayhan** [tinacms/tinacms#3897] [General]
I need to query my posts using a category name. Every post has multiple categories. Someone, please help me.

![image](https://user-images.githubusercontent.com/7611746/235883449-bc4e7ae7-1678-44b3-a9b3-56b39d47c463.png)

I tried to filter by this, but no output

```bash
{
  articlesConnection(filter: {categories: {in: ["featured"]}}) {
    edges {
      node {
        title
        date
        categories
      }
    }
  }
}
```

**Comment by kldavis4** [comment]
@kingRayhan this isn't currently supported but has been added in this pull request: #3901
